README.md

```diff
@@ -463,7 +463,7 @@ Input:
   <a href="https://github.com/intel/cve-bin-tool/blob/main/doc/MANUAL.md#--sbom-file-sbom_file">--sbom-file SBOM_FILE</a>
                         provide sbom filename
   <a href="https://github.com/intel/cve-bin-tool/blob/main/doc/MANUAL.md#d--vex-file-vex_file">--vex-file VEX_FILE</a>
-                        provide vex filename used for triage processing, the type of vex will be automatically detected.
+                        provide vulnerability exchange (vex) filename for triage processing
```

Comment:
should be "vulnerability exploitability exchange"

Reply:
corrected